### DISCUSSION TITLE
Making Swift Arrow cards

### DISCUSSION THREAD

**Discussion by debanjandhar12** [debanjandhar12/logseq-anki-sync#91] [Documentation]
Documentation has been moved from this location to this page: https://debanjandhar12.github.io/logseq-anki-sync/docs/card-types/swift-arrow

**Comment by Tubo** [comment]
For the reverse cards, I've noticed Logseq does a beautiful arrow ligature with `->` and `<-`. I'm wondering have you thought about add the reverse card syntax with `:<-`? 

Update: reading the source code, it looks like `:-<` is a typo. You may want to fix it in the documentations.

**Reply by debanjandhar12** [reply]
Ya, it was a typo. Thanks for informing!

Edit: typo fixed

**Reply by FernandoChu** [reply]
Now I'm very curious, I don't get any ligatures at all, how to enable this?

**Comment by pebblot** [comment]
Hello, really liking your Anki plugin. I want to maintain the balance between primarily using Anki (with help of your plugin) for vocabulary but still being able to use Logseq to learn vocabulary (just in case).
My vocabulary card looks like this:

> [[Question]] :<-> {{cloze answer}} #card #topic #vocabulary

However, using the cloze function, creates a secondary anwer card: 
1. question = answer
2. question = {{cloze answer}}

While using the command ":<->", is it possible to disable the additional card due to the "cloze"-command?
And changing my method this way, is unfortunately not an option for me, because I want to create a [[page]] for each vocab

> {{cloze answer}} : {{cloze answer}} #card #topic #vocabulary

What do you recommend? 
Thank you.
